go.mod

```diff
@@ -3,10 +3,9 @@ module github.com/square/sharkey
 require (
 	bitbucket.org/liamstask/goose v0.0.0-20150115234039-8488cc47d90c
 	github.com/armon/go-metrics v0.3.4
-	github.com/bradleyfalzon/ghinstallation v1.1.1
+	github.com/bradleyfalzon/ghinstallation/v2 v2.0.3
```

Comment:
I thought that was a pretty big jump, but actual v1-to-v2 changes seem pretty small https://github.com/bradleyfalzon/ghinstallation/compare/v1.1.1...v2.0.0